addons/settings/stringtable.xml

```diff
@@ -406,12 +406,30 @@
             <Chinesesimp>此项设定被客户端给覆写并仅适用于伺服器。</Chinesesimp>
             <Portuguese>Esta configuração é sobrescrita pelos clientes e é apenas aplicada no servidor.</Portuguese>
             <Russian>Этот параметр перезаписан клиентом и применяется исключительно к серверу.</Russian>
-            <Polish>Ta wartość zostałą nadpisana przez klientów i ma zastosowanie tylko na serwerze.</Polish>
+            <Polish>Ta wartość została nadpisana przez klientów i ma zastosowanie tylko na serwerze.</Polish>
             <French>Ce paramètre est écrasé par les clients et ne s'applique qu'au serveur.</French>
             <Turkish>Oyuncular tarafından bu ayarın üstüne yazılmış, ayar sadece sunucuda geçerli.</Turkish>
             <Italian>Questa impostazione è sovrascritta dai client e si applica solo al server.</Italian>
             <Czech>Toto nastavení je přepsáno klienty a platí pouze na serveru.</Czech>
         </Key>
+        <Key ID="STR_cba_settings_overwritten_by_client_equal_tooltip">
+            <English>Overwritten by clients with the same value.</English>
```

Comment:
You don't know that. You only know about your client (singular, which is why I told you to change to singular in German too).